### PR TITLE
[MASTER] docs: using GTM

### PR DESCRIPTION
**Pull request by IngridCavalcantiGti**
Documentação da implementação do data-gtm

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1657,3 +1658,83 @@ const Person = ({
...
+new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+})(window,document,'script','dataLayer','GTM-PGRSV4S');</script>
+<!-- End Google Tag Manager -->
+```
+Also, paste this code after the opening `<body>` tag
+```ts
+<!-- Google Tag Manager (noscript) -->
+<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-PGRSV4S"
```
Comment (JulioCVaz) [inline README.md:1691]: (question): this `GTM-PGRSV4S` is our account id?

File: README.md [hunk 2]
```diff
@@ -1657,3 +1658,83 @@ const Person = ({
   // ...
 };
 ```
+## 12. Google Tag Manager
+- 12.1 [What is Google Tag Manager](#12.1-what-is-google-tag-manager)
+- 12.2 [Installations](#12.2-installations)
```
Comment (tassioFront) [inline README.md:1663]: Installation might changes depends on the project's stacks. We can remove it.

Comment (IngridCavalcantiGti) [reply README.md:1663]: @tassioFront  do you think it's better to update this later, in case it changes?

Comment (tassioFront) [reply README.md:1663]: @IngridCavalcantiGti, to be honest, I would remove it. On the LV, for example, we do not use this approach.

Comment (IngridCavalcantiGti) [reply README.md]: done @tassioFront  [ffb02f3](https://github.com/juntossomosmais/frontend-guideline/pull/41/commits/ffb02f341eadbc9da4095cee2155c1777cca8069)

File: README.md [hunk 3]
```diff
@@ -1717,10 +1694,10 @@ After finishing all the settings in the GTM container, we click on "Send" in the
 We can go back to the Analytics tab, under the "Workspace" section and check the tag code termination. We see, in the upper right corner of the screen:
 `GTM-PGRSV4S`
```
Comment (tassioFront) [inline README.md:1695]: Suggestion: We see, in the upper right corner of the screen a code with the following structure: GTM-XXXXXXX

ID might change and also it isn't well to put ids in public docs.

Comment (IngridCavalcantiGti) [reply README.md]: @tassioFront  done [1277799](https://github.com/juntossomosmais/frontend-guideline/pull/41/commits/1277799d6186b46764e821d83faa459c1da839ca)

File: README.md (hunk 1 above)
Comment (IngridCavalcantiGti) [reply README.md]: @JulioCVaz  I removed the installation [ffb02f3](https://github.com/juntossomosmais/frontend-guideline/pull/41/commits/ffb02f341eadbc9da4095cee2155c1777cca8069)

**Review by tassioFront — DISMISSED** [review]

**Review by tassioFront — APPROVED** [review]

**Review by vmcustodio — APPROVED** [review]

**Review by JulioCVaz — APPROVED** [review]

**Pull request merged**